docs/01-Introduction.md

```diff
@@ -1,51 +1,62 @@
 # 1.0 - Introduction
 
 ### 1.1 What is LORIS-MRI? 
-LORIS-MRI is the backbone of the imaging component that makes up LORIS.
- It is maintained in a separate repository so that it can be installed
- on the file server and separated from the web server.
+
+LORIS-MRI comprises the core libraries for loading and inserting imaging 
+data in LORIS. 
+It is maintained in a separate repository so that it can be installed 
+on the file server and separated from the web server.
 These documents assume you have some knowledge regarding LORIS and
-a functioning installation. For information regarding LORIS itself,
-please consult the [LORIS wiki][1].
-
-LORIS-MRI is a set of libraries responsible for the insertion,
-organization, and archival of uploaded imaging datasets.
- It expects an uploaded, compressed file containing
-a [DICOM][2] study composed of several DICOM files. These DICOM files
- will be archived on the server and converted to [MINC][3] and (optionally)
+a functioning installation of the core LORIS codebase. For information 
+regarding LORIS itself, please consult the [LORIS wiki][1].
+
+LORIS-MRI is a set of libraries, scripts, and settings responsible for the 
+insertion, organization, and archiving of uploaded imaging datasets. 
+It expects an uploaded, compressed file containing 
+a [DICOM][2] scan session composed of many DICOM files. These DICOM files 
+will be archived on the server and converted to [MINC][3] and (optionally)
 [NIfTI][4] files. Knowledge of these file formats can be helpful, but are not 
 necessary for using or installing LORIS-MRI.
 
 LORIS-MRI allows you to easily organize and archive your imaging datasets
-and links them with
- corresponding behavioral data in LORIS. Scans can be viewed and quality
- controlled through the user's web browser, facilitating collaboration
- between radiologists, clinicians and researchers.
+and links them with corresponding behavioral data in LORIS. Scans can be viewed 
+and quality controlled in the LORIS front end via web browser, facilitating 
+collaboration between radiologists, clinicians and researchers.
+
+### 1.2 How does LORIS-MRI work? 
+ 
+![user_story](images/user_story.png)
```

Comment:
@davidblader do you still have the original figure? Can we make a slight modification to this? If the `DicomArchive` fails, it also changes the status to `Failure`. Would you be able to add an additional arrow pointing from the `Dicom Archive` to the `Upgrade Progress of entry in Imaging Uploader to 'Failure'`?

Reply:
@davidblader 
ping ping...

Reply:
done